packages/compile-common/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "types": "dist/src/index.d.ts",
   "dependencies": {
-    "@truffle/error": "^0.1.0",
+    "@truffle/error": "0.1.0",
```

Comment:
Is there some reason you pinned these...?  Our dependencies on our own packages aren't normally pinned.

Reply:
yeah we should probably `^` this - sorry, `lernaupdate` pins it by default and I neglected to doctor it up